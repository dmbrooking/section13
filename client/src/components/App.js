import React from 'react'
import NavBar from '../containers/NavBarContainer'

class App extends React.Component {
  render () {
    const { children } = this.props
    return (
      <div>
        <NavBar />
        <div className='container' style={{ marginTop: '50px' }}>
          <div className='row'>
            <div className='col-md-12'>
              { children }
            </div>
          </div>
        </div>
      </div>
    )
  }
}

export default App
