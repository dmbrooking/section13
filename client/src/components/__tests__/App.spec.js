import React from 'react'
import configureStore from 'redux-mock-store'

import App from '../App'

describe('<App />', () => {
  const mockStore = configureStore()
  let store, wrapper

  beforeEach(() => {
    store = mockStore({})
    wrapper = shallow(<App store={store} />)
  })

  it('matches saved snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
