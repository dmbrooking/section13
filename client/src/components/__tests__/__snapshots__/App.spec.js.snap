// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches saved snapshot 1`] = `
<div>
  <Connect(Connect(NavBar)) />
  <div
    className="container"
    style={
      Object {
        "marginTop": "50px",
      }
    }
  >
    <div
      className="row"
    >
      <div
        className="col-md-12"
      />
    </div>
  </div>
</div>
`;
